frontend/components/MainPost.tsx

```diff
@@ -0,0 +1,109 @@
+import styled from 'styled-components';
+import EiditorSkeleton from '@/components/editor/EiditorSkeleton';
+import dynamic from 'next/dynamic';
+import { ChangeEvent } from 'react';
+import { COMMUNITY_EX, PROJECT_EX } from '@/constant/constant';
+const Editor = dynamic(() => import('@/components/editor/Editor'), {
+  ssr: false,
+  loading: () => <EiditorSkeleton />,
+});
+
+type Props = {
+  type: number;
+  form: {
+    tagVal?: string;
+    jobVal?: string;
+    position: string;
+    formTitle: string;
+  };
+  changeEditor: (value: any) => void;
+  onChange: (e: ChangeEvent<HTMLInputElement>) => void;
+};
+
+const MainPost = ({ form, changeEditor, onChange, type }: Props) => {
+  return (
+    <Main>
+      {type === 1 ? PROJECT_EX : COMMUNITY_EX}
+      <form action="#">
+        <div className="nanum-bold">
+          <div>나의 포지션</div>
+          <div>
+            <input
+              name="position"
+              value={form.position}
+              onChange={onChange}
+              type="text"
+              placeholder="포지션을 입력해주세요."
+            />
```

Comment:
`react-hook-form` 을이용하면 여러 인풋을 관리하기 편해집니다.
상단에`react-hook-form`을  추가합니다.
`import { useForm } from 'react-hook-form';`
유즈폼을 이용합니다.
`const { register, watch, handleSubmit } = useForm<ILoginForm>();`
이제 단순히 컴포넌트 내에 `watch`를 추가하는 행동만으로 입력 내용을 실시간으로 모니터링 할 수 있습니다.
`console.log(watch())`
동시에 여러 input에 적용가능합니다.

```suggestion
            <input
              {...register("position")}
              type="text"
              placeholder="포지션을 입력해주세요."
            />
```

Reply:
좋은 리뷰 감사드립니다!
한 번도 사용해본적 없는 라이브러리라 몰랐는데 정말 간단해지군요 ㄷㄷ 
리팩토링 시도해보겠습니다.